blockstore/splitstore/splitstore_compact.go

```diff
@@ -1047,123 +1077,129 @@ func (s *SplitStore) walkChain(ts *types.TipSet, inclState, inclMsgs abi.ChainEp
 		}
 	}
 
-	log.Infow("chain walk done", "walked", *walkCnt, "scanned", *scanCnt)
-
+	log.Infow("chain walk done", "walked", *walkCnt, "scanned", *scanCnt, "walk size", szWalk)
+	s.szWalk = *szWalk
```

Comment:
So while there is a 99% chance that this is correct, I'd still use atomic.LoadInt to get the value, just out of abundance of caution around thread-shared variables